tests/rados/test_osd_scrub_load_threshold_parameter.py

```diff
@@ -0,0 +1,548 @@
+"""
```

Comment:
The current workflow only tests with Scheduled scrubs.

As part of next PR, enhance the workflow with both user & scheduled scrubs.

Reply:
Added the sprint task - https://issues.redhat.com/browse/RHCEPHQE-21446